docs/Clearwater_SNMP_Statistics.md

```diff
@@ -110,6 +110,7 @@ Sprout nodes provide the following statistics:
 * The number of Shared iFC set IDs retrieved from the HSS where the S-CSCF has no matching set of iFCs, indexed by time period.
 * The number of initial requests that had no matching iFCs to apply, indexed by time period.
 * The number of initial requests that attempted to use Default iFCs but had no matching ones, indexed by time period.
+* The number of requests rejected because an IMPU was barred.
```

Comment:
I would match this to what you said in:
https://github.com/Metaswitch/clearwater-snmp-handlers/pull/186#pullrequestreview-40985426
What you said there was "Number of requests rejected because an identity was barred."
(alternatively change that to match this?)